go.mod

```diff
@@ -26,6 +30,7 @@ require (
 	github.com/riferrei/srclient v0.5.4
 	github.com/shopspring/decimal v1.4.0
 	github.com/smartcontractkit/libocr v0.0.0-20240419185742-fd3cab206b2c
+	github.com/spf13/cobra v1.8.1
```

Comment:
we are using `github.com/urfave/cli` in core. let's stay unified.

Reply:
not sure what pulled this in.

Reply:
It's not a drop in replacement and we use it in the tools (see cmd for example).  Is this a blocker for you or can it be done later?

Reply:
The problem is that this repo is imported everywhere else, so these kinds of deps have major consequences. What features are we using? Could we even use pure std lib instead?

Reply:
What does the standard provide?  Can this be done in a follow-up PR instead (lower priority), I really want to unblock the rest of my work.

Reply:
If you just need basic [flags](https://pkg.go.dev/flag) and [env vars](https://pkg.go.dev/os), then there is no need for a third party library. Are you using any other features?

Reply:
@DeividasK you set up the CLI, mind looking at ^